app/src/main/java/com/kickstarter/viewmodels/CommentsViewModel.kt

```diff
@@ -154,9 +154,11 @@ interface CommentsViewModel {
                 .compose(Transformers.takeWhen(this.refresh))
                 .doOnNext {
                     this.isRefreshing.onNext(true)
-                    // reset cursor
+                }
+                .doOnNext {
```

Comment:
You have two `doOnNext`, look at line 155, you can combine them

Reply:
done